src/components/test-results/test-results.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactEcharts from "echarts-for-react";
+import CaretSvg from "../../images/icons/caret-down.svg";
+import "./test-results.styles.scss";
+import { Label } from "../label/label";
+import format from "date-fns/format";
+import fromUnixTime from "date-fns/fromUnixTime";
+
+export const TestResults = ({ personalData = {}, results = [] }) => {
```

Comment:
The name is not very intuitive. Plus, when used in a different context it might not be "test results" (In the Dashboard for example). The table could be name just `Table` or something similar, and the chart something like `BarChart`